web/packages/design/src/TextArea/TextArea.tsx

```diff
@@ -28,7 +28,7 @@ export interface TextAreaProps extends React.ComponentPropsWithRef<'textarea'> {
 
 export const TextArea: React.FC<TextAreaProps> = styled.textarea`
```

Comment:
What's the reason behind changes to TextArea? These strike me as substantial and they're not included in the PR description.

Reply:
For some reason, when I originally added `TextArea` I didn't include all these styles in that component. It feels weird to add them in every place where `TextArea` is used. 
Now `TextArea` is more similar to `Input` which includes all styles.
Also, this component is used only in our feedback form.